### PR TITLE
set custom user agent

### PR DESCRIPTION
**Pull request by philipkozeny**
fixes https://github.com/mediagis/nominatim-docker/issues/413

### CODE REVIEW

**Comment by leonardehrenfried** [comment]
It seems that the user agent isn't passed along to the bash scripts correctly. I had a similar problem once with the password and if I remember correctly it was fixed by moving the variable declaration towards the end of the Dockerfile. I'm speculating that envs aren't transferred when collapsing the layers.

**Comment by leonardehrenfried** [comment]
The thing is that you need it when building the image... Not sure if there is a solution where you don't declare it twice.

**Review by leonardehrenfried — APPROVED** [review]

File: 4.2/init.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ OSMFILE=${PROJECT_DIR}/data.osm.pbf
 
 if [ "$IMPORT_WIKIPEDIA" = "true" ]; then
   echo "Downloading Wikipedia importance dump"
-  curl https://nominatim.org/data/wikimedia-importance.sql.gz -L -o ${PROJECT_DIR}/wikimedia-importance.sql.gz
+  curl -A {$USER_AGENT} https://nominatim.org/data/wikimedia-importance.sql.gz -L -o ${PROJECT_DIR}/wikimedia-importance.sql.gz
```
Comment (lonvia) [inline 4.2/init.sh:7]: Was that meant to be `-A ${USER_AGENT}`? Same below.

Comment (philipkozeny) [reply 4.2/init.sh:7]: Oh yes ... thanks for the catch!

**Pull request merged**